src/user-research/how-we-do-user-research.md

```diff
@@ -27,28 +27,29 @@ These are the things that user researchers usually do on projects, and the relev
    [Setting up a project](/delivery-management/setting-up-a-new-project/) \
    [Creating and using research plans](/user-research/creating-and-using-research-plans/)
 
-2. **Create and share research plans**
+1. **Create and share research plans**
 
    Researchers create and share research plans that show what we’re aiming to learn, which groups we’ll be researching with, and what activities we will do. We involve our teams in creating these plans, and share them widely, so our colleagues can understand and support the research we will be doing.
 
-   We choose research activities that provide strong evidence and reliable answers, for the least time, effort and cost. And we choose activities that help our teams and different kinds of users to fully participate. We make sure the research is ethical. And that the plan is achievable and allows us to do research at a sustainable pace.
+   We choose research activities that provide strong evidence and reliable answers, for the least time, effort and cost. And we choose activities that help our teams and different kinds of users to fully participate. We make sure the research is ethical and safe. And that the plan is achievable and allows us to do research at a sustainable pace.
 
    This helps us to produce the clearest and most valuable findings we can, with the people, time and other resources we have.
 
    [Creating and using research plans](/user-research/creating-and-using-research-plans/) \
-   [Choosing and using research methods](/user-research/choosing-and-using-research-methods/)
+   [Choosing and using research methods](/user-research/choosing-and-using-research-methods/) \
+   [Doing research safely](/user-research/doing-research-safely)
 
-3. **Recruit people for research**
+1. **Recruit people for research**
 
    Researchers recruit appropriate participants for research activities. Finding the right participants at the right time is always challenging, so we start recruiting as soon as we can and use a variety of approaches to reach everyone we need. We keep track of who participates in our research, so we can be confident that we’re including the right people. And we work hard to protect their privacy and wellbeing.
 
    This helps teams to learn about all the people who might use or be affected by a service.
 
+   [Recruiting research participants and protecting their privacy](/user-research/recruiting-participant-and-protecting-privacy/) \
```

Comment:
`participant` -> `participants` in the link

Reply:
Thanks for spotting